js/src/config.ts

```diff
@@ -14,6 +14,14 @@ const liquidityMiningConfig = {
         fleekNamespace: 'balancer-claims-lido',
         offset: 63,
     },
+    'homestead-vita': {
+        reportsDirectory: `../reports/`,
+        reportFilename:
+            '__ethereum_0x81f8f0bb1cB2A06649E51913A151F0E7Ef6FA321.json',
```

Comment:
The report filename is lowercase `__ethereum_0x81f8f0bb1cb2a06649e51913a151f0e7ef6fa321.json`